tracee/tracee.go

```diff
@@ -570,55 +581,9 @@ type context struct {
 	Retval  int64
 }
 
-func (t *Tracee) processEvents() {
+func (t *Tracee) processLostEvents() {
 	for {
 		select {
```

Comment:
what is the purpose of the select here? previously it selected between processing an event an a lostevent, now I think it's redundant.